spec/usermanual/teachers.html.erb

```diff
@@ -93,6 +93,7 @@ require 'cgi' # for escapeHTML
     <ul>
       <li><b>Name</b> of the organization that will be seen in the app</li>
       <li><b>Description</b> about your organization, seen in this organization's page</li>
+      <li><b>Logo</b> for your organization, seen in this organization's page (you can change this and other fields above later)</li>
```

Comment:
Current size of the logo is 100*100. I think most of the logos are bigger than that. Should we still restrict acceptance of images at least that 100*100 size?